src/modules/utils/globals.js

```diff
@@ -23,9 +23,10 @@ export const editor = () => tec().editor || {};
 export const settings = () => tec().settings || {};
 export const mapsAPI = () => tec().googleMap || {};
 export const priceSettings = () => tec().priceSettings || {};
+export const tecDateSettings = () => tec().dateSettings || {};
```

Comment:
Looks good however it might be simpler to call `tec()` where is used instead of creating a new variable for that setting.

Reply:
every other setting is using this convention so i've followed that. also it's handy to have this as if tec isn't activated, it returns a blank object and the property is undefined (makes truthy/falsey checking easy)